scan/src/pages/ValidatorIndexPage.re

```diff
@@ -75,100 +75,125 @@ let kvRow = (k, v: value_row_t) => {
-          <Text
-            value="VALIDATOR DETAILS"
-            weight=Text.Medium
-            size=Text.Md
-            spacing={Text.Em(0.06)}
-            height={Text.Px(15)}
-            nowrap=true
-            color=Colors.gray7
-            block=true
-          />
-          <div className=Styles.seperatedLine />
-          <Text
-            value={isActive ? "ACTIVE" : "INACTIVE"}
-            size=Text.Md
-            weight=Text.Thin
-            spacing={Text.Em(0.06)}
-            color=Colors.gray7
-            nowrap=true
-          />
-          <HSpacing size=Spacing.md />
-          <img
-            src={isActive ? Images.activeValidatorLogo : Images.inactiveValidatorLogo}
-            className=Styles.logoSmall
-          />
+          <Text value="CoinGecko Data Provider" size=Text.Xxl weight=Text.Bold nowrap=true />
         </div>
-      </Col>
-    </Row>
-    <VSpacing size=Spacing.xl />
-    <div className=Styles.vFlex>
-      <Text value="CoinGecko Data Provider" size=Text.Xxl weight=Text.Bold nowrap=true />
-    </div>
-    <VSpacing size=Spacing.xl />
-    <div className=Styles.topPartWrapper>
-      <Text value="INFORMATION" size=Text.Lg weight=Text.Semibold />
-      <VSpacing size=Spacing.lg />
-      {kvRow("OPERATOR ADDRESS", VValidatorAddress(address))}
-      <VSpacing size=Spacing.lg />
-      {kvRow("ADDRESS", VAddress(address))}
-      <VSpacing size=Spacing.lg />
-      {kvRow("VOTING POWER", VCode("45.34% (12,434.234 BAND)"))}
-      <VSpacing size=Spacing.lg />
-      {kvRow("COMMISSION", VCode("3.00%"))}
-      <VSpacing size=Spacing.lg />
-      {kvRow("BONDED HEIGHT", VCode("1"))}
-      <VSpacing size=Spacing.lg />
-      {kvRow("WEBSITE", VExtLink("https://coingecko.node"))}
-      <VSpacing size=Spacing.lg />
-      {kvRow(
-         "DETAILS",
-         VText("We are the leading staking service provider for blockchain projects."),
-       )}
-      <div className=Styles.longLine />
-      <div className={Styles.fullWidth(`row)}>
-        <Col size=1.>
-          <Text value="NODE STATUS" size=Text.Lg weight=Text.Semibold />
+        <VSpacing size=Spacing.xl />
+        <div className=Styles.topPartWrapper>
+          <Text value="INFORMATION" size=Text.Lg weight=Text.Semibold />
           <VSpacing size=Spacing.lg />
-          {kvRow("UPTIME", VCode("100.00%"))}
+          {kvRow("OPERATOR ADDRESS", VValidatorAddress(address))}
           <VSpacing size=Spacing.lg />
-          {kvRow("AVG. RESPONSE TIME", VCode("2 blocks"))}
-        </Col>
-        <HSpacing size=Spacing.lg />
-        <Col size=1.>
-          <Text value="REQUEST RESPONSE" size=Text.Lg weight=Text.Semibold />
+          {kvRow("ADDRESS", VAddress(address))}
           <VSpacing size=Spacing.lg />
-          {kvRow("COMPLETED REQUESTS", VCode("23,459"))}
+          {kvRow(
+             "VOTING POWER",
+             VCode(
+               (validator.votingPower /. marketcap)->Format.fPretty
```

Comment:
`% of votingPower` is calculated by `votingPower of the validator / allBondedAmount`

Reply:
validator.votingPower *. 100. /. totalPower